app/controllers/journals_controller.rb

```diff
@@ -110,17 +84,6 @@ def diff
     end
   end
 
-  def preview
-    @journal.notes = params[:notes]
-
-    respond_to do |format|
-      format.any(:html, :js) do
-        render locals: { journal: @journal },
-               layout: false
-      end
-    end
-  end
-
   private
 
   def find_journal
```

Comment:
I've rewritten the routes (and integrates the diff route while doing so), and removed `default_breadcrumb`, as the other methods are still required.